pkg/util/platform.go

```diff
@@ -0,0 +1,30 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/containerd/containerd/platforms"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+func normalizeContainerPlatfrom(platform v1.Platform) v1.Platform {
+	platform.OS = "linux"
+	return platforms.Normalize(platform)
+}
```

Comment:
why setting this to linux?

Reply:
Same as above. If you run `docker buildx ls`, you'll get a list of all supported runtimes by your machine; regardless of systems architecture, docker runtimes are all `linux`.